src/DataCollection.UWP/Views/OriginRelatedRecordPopup.xaml.cs

```diff
@@ -74,5 +75,21 @@ private async void OnPropertyChanged([CallerMemberName] string propertyName = nu
         }
 
         public event PropertyChangedEventHandler PropertyChanged;
+
+        /// <summary>
+        /// Event handler for user selecting to add a new attachment by capturing new media
+        /// </summary>
+        private async void CaptureMediaButton_Click(object sender, Windows.UI.Xaml.RoutedEventArgs e)
```

Comment:
Could these be refactored to be commands instead of using click event handlers?

Reply:
Looking at this closer, I see why maybe it isn't as straightforward to do this - the behavior and expectations vary significantly between platforms, enough that is probably doesn't make sense to move that to shared code.